src/context.cpp

```diff
@@ -699,6 +700,40 @@ namespace librealsense
             software_sensor::open( profiles );
         }
 
+        void custom_on_video_frame( rs2_software_video_frame const & software_frame )
```

Comment:
Why not override the base class `on_video_frame`?
`custom` does not convey any meaning. If not overriding use `on_video_frame_deffered_deleter`

Reply:
Because eventually we won't even be using `rs2_software_video_frame`. We can maybe still override the name, but let's do it separately -- for now I like that it's clear that this is not some virtual function (which it isn't) that can be called from elsewhere.